packages/core-modules/contracts/interfaces/IElectionModule.sol

```diff
@@ -28,7 +28,7 @@ interface IElectionModule {
 
     function setNextPeriodPercent(uint8 percent) external;
 
-    function elect(address[] memory candidates) external;
+    function elect(address[] memory sortedCandidates, uint[] memory positions) external;
```

Comment:
This interface looks confusing IMO.

`sortedCandidates` 👉 sorted how? Numerically or politically?
It might be too long, but `numericallySortedCandidates` might be more semantically correct.

`positions` 👉 positions of what?
Again, I'd use something more semantically aligned. Maybe `priorities`.

Reply:
👍 , updated to `numericallySortedCandidates` & `priorities`